src/openshift/component.ts

```diff
@@ -28,16 +28,43 @@ export class Component extends OpenShiftItem {
         }
     }
 
+    static async delComponent () {
```

Comment:
Could you change it to work the same way as I implemented for `OpenShift: Delete Application`. It is much easier right now.